templates/server/src/main/resources/archetype-resources/core/src/main/java/__packageInPathFormat__/general/logic/impl/UcManageBinaryObjectImpl.java

```diff
@@ -48,20 +48,20 @@ public BinaryObjectEto saveBinaryObject(Blob data, BinaryObjectEto binaryObjectE
   @Override
   public void deleteBinaryObject(Long binaryObjectId) {
 
-    this.binaryObjectRepository.delete(binaryObjectId);
+    this.binaryObjectRepository.deleteById(binaryObjectId);
```

Comment:
Woah. So spring-data changed their API in an incompatible way? That is not so great to here. I will double check...

Reply:
Verified. Nothing we can do about it but good that we have these upgrades before we actually spread out spring-data support in devon.